Source/WebCore/inspector/InspectorFrontendHost.cpp

```diff
@@ -408,12 +408,12 @@ String InspectorFrontendHost::platform() const
 
 String InspectorFrontendHost::platformVersionName() const
 {
-#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000
+#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000
+    return "ventura"_s;
```

Comment:
This is going to break styling in the Web Inspector frontend (WebInspectorUI project) as is. We currently share style between Monterey and Ventura, so didn't update this. Searching for `monterey` in `WebInspectorUI` will reveal all the places we assume the modern platform style will be named `monterey`.